### PR TITLE
configurable parameter added to manage ga fallback for cid

### PR DESCRIPTION
**Pull request by Tnmy1991**
## Description of the change

configurable parameter added to manage ga fallback for cid

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/ga/transform.js [hunk 1]
```diff
@@ -299,11 +299,17 @@ function responseBuilderSimple(
       ? message.integrations[gaDisplayName].clientId
       : undefined
     : undefined;
-  finalPayload.cid =
-    integrationsClientId ||
-    getDestinationExternalID(message, "gaExternalId") ||
-    message.anonymousId ||
-    md5(message.userId);
+
+  const disableMd5 = get(message, "disableMd5");
```
Comment (arnab-p) [inline v0/destinations/ga/transform.js:303]: IMO, it's better to control it through a dashboard setting. WDYT @ruchiramoitra?

Comment (ruchiramoitra) [reply v0/destinations/ga/transform.js:303]: yes. we can take it from destination config.

File: v0/destinations/ga/transform.js [hunk 2]
```diff
@@ -299,11 +299,17 @@ function responseBuilderSimple(
...
+  if (disableMd5) {
+    finalPayload.cid = undefined;
```
Comment (ruchiramoitra) [inline v0/destinations/ga/transform.js:305]: we shouldn't keep this undefined. if md5 is disabled 
cid =  finalPayload.cid =
      integrationsClientId ||
      getDestinationExternalID(message, "gaExternalId") ||
      message.anonymousId ||
     message.userId;

Comment (Tnmy1991) [reply v0/destinations/ga/transform.js:305]: Okay, got it

File: v0/destinations/ga/transform.js (hunk 1 above)
Comment (Tnmy1991) [reply v0/destinations/ga/transform.js:303]: Okay, update soon

Comment (Tnmy1991) [reply v0/destinations/ga/transform.js]: changes are updated

File: v0/destinations/ga/transform.js (hunk 2 above)
Comment (Tnmy1991) [reply v0/destinations/ga/transform.js]: changes are updated

**Comment by arnab-p** [comment]
Related PRs - 

https://github.com/rudderlabs/rudder-webapp/pull/656
https://github.com/rudderlabs/rudder-config-backend/pull/613

**Review by arnab-p — APPROVED** [review]

**Review by ruchiramoitra — APPROVED** [review]

**Pull request merged**